creator-node/src/recurringSync.js

```diff
@@ -0,0 +1,31 @@
+const Bull = require('bull')
+const { logger } = require('./logging')
+const config = require('./config.js')
+
+class RecurringSync {
+  constructor () {
+    this.recurringSyncQueue = new Bull(
+      'recurring-syncs',
+      {
+        redis: {
+          port: config.get('redisPort'),
+          host: config.get('redisHost')
+        }
+      }
+    )
+  }
+  async init () {
+    await this.recurringSyncQueue.empty()
+
+    // run the task every 6 hours
+    this.recurringSyncQueue.add({}, { repeat: { cron: '0 */6 * * *' } })
```

Comment:
@dmanjunath did you validate this fires at a smaller interval?

Reply:
yeah i did every minute and it fired

Reply:
logs from run with cron set to every minute

```
[2020-06-04T21:28:00.013Z]  INFO: audius_creator_node/63718 on Dheerajs-MBP-2.lan: Process recurring sync
[2020-06-04T21:28:14.088Z] DEBUG: audius_creator_node/63718 on Dheerajs-MBP-2.lan: Begin processing request (requestID=NsHuh2oLP, requestMethod=GET, requestHostname=docker.for.mac.localhost, requestUrl=/ipfs_peer_info)
[2020-06-04T21:28:14.093Z]  INFO: audius_creator_node/63718 on Dheerajs-MBP-2.lan: Success (requestID=NsHuh2oLP, requestMethod=GET, requestHostname=docker.for.mac.localhost, requestUrl=/ipfs_peer_info, statusCode=200)
[2020-06-04T21:28:14.094Z]  INFO: audius_creator_node/63718 on Dheerajs-MBP-2.lan: Request Duration 5.589ms (requestID=NsHuh2oLP, requestMethod=GET, requestHostname=docker.for.mac.localhost, requestUrl=/ipfs_peer_info)
[2020-06-04T21:28:29.060Z] DEBUG: audius_creator_node/63718 on Dheerajs-MBP-2.lan: Begin processing request (requestID=T573FRMLf, requestMethod=GET, requestHostname=docker.for.mac.localhost, requestUrl=/ipfs_peer_info)
[2020-06-04T21:28:29.065Z]  INFO: audius_creator_node/63718 on Dheerajs-MBP-2.lan: Success (requestID=T573FRMLf, requestMethod=GET, requestHostname=docker.for.mac.localhost, requestUrl=/ipfs_peer_info, statusCode=200)
[2020-06-04T21:28:29.065Z]  INFO: audius_creator_node/63718 on Dheerajs-MBP-2.lan: Request Duration 5.136ms (requestID=T573FRMLf, requestMethod=GET, requestHostname=docker.for.mac.localhost, requestUrl=/ipfs_peer_info)
[2020-06-04T21:28:44.085Z] DEBUG: audius_creator_node/63718 on Dheerajs-MBP-2.lan: Begin processing request (requestID=DTAHYrkcK, requestMethod=GET, requestHostname=docker.for.mac.localhost, requestUrl=/ipfs_peer_info)
[2020-06-04T21:28:44.092Z]  INFO: audius_creator_node/63718 on Dheerajs-MBP-2.lan: Success (requestID=DTAHYrkcK, requestMethod=GET, requestHostname=docker.for.mac.localhost, requestUrl=/ipfs_peer_info, statusCode=200)
[2020-06-04T21:28:44.092Z]  INFO: audius_creator_node/63718 on Dheerajs-MBP-2.lan: Request Duration 7.408ms (requestID=DTAHYrkcK, requestMethod=GET, requestHostname=docker.for.mac.localhost, requestUrl=/ipfs_peer_info)
[2020-06-04T21:28:59.079Z] DEBUG: audius_creator_node/63718 on Dheerajs-MBP-2.lan: Begin processing request (requestID=LRbHsuORM, requestMethod=GET, requestHostname=docker.for.mac.localhost, requestUrl=/ipfs_peer_info)
[2020-06-04T21:28:59.085Z]  INFO: audius_creator_node/63718 on Dheerajs-MBP-2.lan: Success (requestID=LRbHsuORM, requestMethod=GET, requestHostname=docker.for.mac.localhost, requestUrl=/ipfs_peer_info, statusCode=200)
[2020-06-04T21:28:59.085Z]  INFO: audius_creator_node/63718 on Dheerajs-MBP-2.lan: Request Duration 6.927ms (requestID=LRbHsuORM, requestMethod=GET, requestHostname=docker.for.mac.localhost, requestUrl=/ipfs_peer_info)
[2020-06-04T21:29:00.016Z]  INFO: audius_creator_node/63718 on Dheerajs-MBP-2.lan: Process recurring sync
[2020-06-04T21:29:14.080Z] DEBUG: audius_creator_node/63718 on Dheerajs-MBP-2.lan: Begin processing request (requestID=NVdcWhbmb, requestMethod=GET, requestHostname=docker.for.mac.localhost, requestUrl=/ipfs_peer_info)
[2020-06-04T21:29:14.085Z]  INFO: audius_creator_node/63718 on Dheerajs-MBP-2.lan: Success (requestID=NVdcWhbmb, requestMethod=GET, requestHostname=docker.for.mac.localhost, requestUrl=/ipfs_peer_info, statusCode=200)
[2020-06-04T21:29:14.086Z]  INFO: audius_creator_node/63718 on Dheerajs-MBP-2.lan: Request Duration 5.881ms (requestID=NVdcWhbmb, requestMethod=GET, requestHostname=docker.for.mac.localhost, requestUrl=/ipfs_peer_info)
[2020-06-04T21:29:29.063Z] DEBUG: audius_creator_node/63718 on Dheerajs-MBP-2.lan: Begin processing request (requestID=-d28hDGT-, requestMethod=GET, requestHostname=docker.for.mac.localhost, requestUrl=/ipfs_peer_info)
[2020-06-04T21:29:29.068Z]  INFO: audius_creator_node/63718 on Dheerajs-MBP-2.lan: Success (requestID=-d28hDGT-, requestMethod=GET, requestHostname=docker.for.mac.localhost, requestUrl=/ipfs_peer_info, statusCode=200)
[2020-06-04T21:29:29.069Z]  INFO: audius_creator_node/63718 on Dheerajs-MBP-2.lan: Request Duration 5.484ms (requestID=-d28hDGT-, requestMethod=GET, requestHostname=docker.for.mac.localhost, requestUrl=/ipfs_peer_info)
[2020-06-04T21:29:44.084Z] DEBUG: audius_creator_node/63718 on Dheerajs-MBP-2.lan: Begin processing request (requestID=1tBW6EmsM, requestMethod=GET, requestHostname=docker.for.mac.localhost, requestUrl=/ipfs_peer_info)
[2020-06-04T21:29:44.091Z]  INFO: audius_creator_node/63718 on Dheerajs-MBP-2.lan: Success (requestID=1tBW6EmsM, requestMethod=GET, requestHostname=docker.for.mac.localhost, requestUrl=/ipfs_peer_info, statusCode=200)
[2020-06-04T21:29:44.092Z]  INFO: audius_creator_node/63718 on Dheerajs-MBP-2.lan: Request Duration 7.270ms (requestID=1tBW6EmsM, requestMethod=GET, requestHostname=docker.for.mac.localhost, requestUrl=/ipfs_peer_info)
[2020-06-04T21:29:59.091Z] DEBUG: audius_creator_node/63718 on Dheerajs-MBP-2.lan: Begin processing request (requestID=OxpXWEDbW, requestMethod=GET, requestHostname=docker.for.mac.localhost, requestUrl=/ipfs_peer_info)
[2020-06-04T21:29:59.097Z]  INFO: audius_creator_node/63718 on Dheerajs-MBP-2.lan: Success (requestID=OxpXWEDbW, requestMethod=GET, requestHostname=docker.for.mac.localhost, requestUrl=/ipfs_peer_info, statusCode=200)
[2020-06-04T21:29:59.098Z]  INFO: audius_creator_node/63718 on Dheerajs-MBP-2.lan: Request Duration 6.887ms (requestID=OxpXWEDbW, requestMethod=GET, requestHostname=docker.for.mac.localhost, requestUrl=/ipfs_peer_info)
[2020-06-04T21:30:00.014Z]  INFO: audius_creator_node/63718 on Dheerajs-MBP-2.lan: Process recurring sync
```